group-coordinator/src/main/java/org/apache/kafka/coordinator/group/Group.java

```diff
@@ -209,4 +221,90 @@ void validateOffsetFetch(
     default boolean shouldExpire() {
         return true;
     }
+
+    /**
+     * The magic byte used to identify the version of topic hash function.
+     */
+    byte TOPIC_HASH_MAGIC_BYTE = 0x00;
+    XXHash64 LZ4_HASH_INSTANCE = XXHashFactory.fastestInstance().hash64();
```

Comment:
Would it be possible to put those and the new methods to a separate class? Having them in `Group` is weird because it won't be used by all the group types.

Reply:
Move to `org.apache.kafka.coordinator.group.Utils`. Thanks.